pubpub/pubpub
# Prevent downloads when suggestions exist

## Issue(s) Resolved
Resolves #2755
Resolves #2734

## Test Plan
On the review instance:
1. Create a pub or visit an existing one
2. Enter suggestion mode and add suggestions
3. Attempt to download the pub
4. Resolve suggestions
5. Download the pub and confirm the download works
6. Create a release
7. Confirm the pub can be downloaded from the release as well

Then visit a community without the SE flag enabled and confirm that downloads work

## Screenshots (if applicable)
<img width="766" alt="image" src="https://github.com/pubpub/pubpub/assets/473542/ffb62957-72f0-403c-a31a-2a35fddf3f05">

## Optional

### Notes/Context/Gotchas

@gabestein could use your approval/advice on the wording and the button names
